src/dev-center/js/dev-center.js

```diff
@@ -31,6 +31,9 @@ let dropped_items;
 let search_query;
 let originalValues = {};
 
+// At the top of your file, add a global variable to track deployment cancellation
+let globalDeploymentCancelled = false;
```

Comment:
We can probably just call this `deploymentCancelled`, since there's no "local deployment" option in the dev center.